### DISCUSSION TITLE
CHANGELOG.md?

### DISCUSSION THREAD

**Discussion by ccristal** [GitTools/GitVersion#2994] [Ideas]
Hi!
Thanks for putting GitVersion together, it has solved many of my headaches... :-)
Any chance of supporting auto-generation of CHANGELOG.md?
Thanks!

**Comment by asbjornu** [comment]
[GitReleaseManager](https://github.com/GitTools/GitReleaseManager) is a tool that may help you perform releases, create automatic release notes, etc.

**Reply by ccristal** [reply]
Ok, but that only works for GitHub, doesn't it?

**Reply by gep13** [reply]
Currently, yes, but it is possible to create providers for additional systems.  What systems were you interested in using?

**Reply by ccristal** [reply]
That shouldn't be relevant, really... All I was after was something that could produce a nicely formatted CHANGELOG.md from the commit messages in conventional format.

**Reply by asbjornu** [reply]
Then perhaps [Release Drafter](https://github.com/marketplace/actions/release-drafter) may be a better fit, @ccristal?
